operations/IT/helpdesk.md

```diff
@@ -0,0 +1,54 @@
+# Helpdesk
+
+The Mattermost IT team provides technical support to all Mattermost staff members and is dedicated to servicing the technology support needs of all staff by providing detailed resolutions and general system information for common problems.
+
+Through the internal helpdesk solution, staff members are able to request application access, elevated permissions, assistance with troubleshooting software/hardware issues, flag security issues, and ask for help with integrations, as well as submit general inquiries.
+
+## Availability
+
+Helpdesk services are available during the following hours of operation, except during office closures and company observed holidays.
+
+* Monday through Friday **9:00 a.m. - 5:00 p.m. EST**
+
+Support is available outside of working hours in the case of an emergency. Any issue classified as **"urgent"** is considered an emergency. **"high"** level incidents may also fall under scope as an emergency depending on the request.
```

Comment:
```suggestion
Support is available outside of working hours in the case of an emergency. Any issue classified as **"urgent"** is considered an emergency. Incident classified as **"high"** level may also fall under the scope of an emergency depending on the request.
```

Reply:
Hey @justinegeffen should this be "Incidents" vs "Incident".